frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -498,9 +498,7 @@ export const OperandDetails = connectToModel((props: OperandDetailsProps) => {
   const primaryDescriptors = statusDescriptors.filter((descriptor) => isMainDescriptor(descriptor));
 
   const header = (
-    <h2 className="co-section-heading">{`${
-      thisDefinition ? thisDefinition.displayName : kind
-    } Overview`}</h2>
+    <h2 className="co-section-heading">{`${thisDefinition?.displayName || kind} Overview`}</h2>
```

Comment:
Sorry, I missed this before, but this can be

```suggestion
    <h2 className="co-section-heading">{thisDefinition?.displayName || kind} Overview</h2>
```